dask_expr/tests/test_collection.py

```diff
@@ -1669,6 +1669,13 @@ def test_quantile(df):
         lib.Series([19.0, 79.0], index=[0.2, 0.8]),
     )
 
+    with pytest.raises(AssertionError):
+        df.x.quantile(q=[]).compute()
```

Comment:
pandas returns an empty series here. Doubt we care to diverge on such an edge case though.

Reply:
Yeah I thought this was kind of pointless, but I might put up a follow up anyway